internal/route/repo/pull.go

```diff
@@ -637,6 +645,14 @@ func CompareAndPullRequest(c *context.Context) {
 	}
 
 	c.Data["IsSplitStyle"] = c.Query("style") == "split"
+	setTemplateIfExists(c, PULL_REQUEST_TITLE_TEMPLATE_KEY, PullRequestTitleTemplateCandidates)
+
+	if c.Data[PULL_REQUEST_TITLE_TEMPLATE_KEY] != nil {
+		customTitle := c.Data[PULL_REQUEST_TITLE_TEMPLATE_KEY].(string)
+		r := strings.NewReplacer("{{headBranch}}", headBranch,"{{baseBranch}}", baseBranch)
+		c.Data["title"] = r.Replace(customTitle)
```

Comment:
Where does this all lower cased "title" come from?

Reply:
Im not sure that i correct understand your question.

"title" usage located here https://github.com/gogs/gogs/blob/master/templates/repo/issue/new_form.tmpl#L16

Reply:
Gotcha, haha, sorry missed it!